helm/kong-app/values.yaml

```diff
@@ -254,7 +255,8 @@ runMigrations: true
 # Specify Kong configurations
 # Kong configurations guide https://getkong.org/docs/latest/configuration/
 env:
-  database: postgres
+  # Disable database by default.
```

Comment:
That is pretty obvious. The comment could be more useful to me when it would explain why we do this. I have no idea about this whole thing and do not really follow why that is.

Reply:
Ack. The templates here are just a copy of the upstream chart.

So I commented all the defaults I changed. I'll add more detail on why they need to be changed.

Reply:
Done. @xh3b4sd PTAL

Reply:
This is all very new but I added a Giant Swarm prefix to the comments. So it's clear it's our changes. I'll discuss with Puja & Fer when they are back.